comptests/motr/test_motr_di.py

```diff
@@ -107,36 +132,128 @@ def m0cp_corrupt_data_m0cat(self, layout_ids, bsize_list, count_list, offsets):
+        validate the corruption with md5sum after M0CAT.
+        """
+        logger.info("STARTED: m0cp, corrupt and m0cat workflow")
+        infile = TEMP_PATH + "input"
+        outfile = TEMP_PATH + "output"
         node_pod_dict = self.motr_obj.get_node_pod_dict()
         motr_client_num = self.motr_obj.get_number_of_motr_clients()
-        object_id = str(self.system_random.randint(1, 1024 * 1024)) + ":" + \
-                    str(self.system_random.randint(1, 1024 * 1024))
+        object_id = (
+            str(self.system_random.randint(1, 1024 * 1024))
+            + ":"
+            + str(self.system_random.randint(1, 1024 * 1024))
+        )
         for client_num in range(motr_client_num):
             for node in node_pod_dict:
 
-                for b_size, (cnt_c, cnt_u), layout, offset in zip(bsize_list, count_list,
-                                                                  layout_ids, offsets):
-                    self.motr_obj.dd_cmd(
-                        b_size, cnt_c, infile, node)
-                    self.motr_obj.cp_cmd(
-                        b_size, cnt_c, object_id,
-                        layout, infile, node, client_num)
+                for b_size, (cnt_c, cnt_u), layout, offset in zip(
+                    bsize_list, count_list, layout_ids, offsets
+                ):
+                    self.motr_obj.dd_cmd(b_size, cnt_c, infile, node)
+                    self.motr_obj.cp_cmd(b_size, cnt_c, object_id, layout, infile, node, client_num)
                     self.motr_obj.cat_cmd(
-                        b_size, cnt_c, object_id,
-                        layout, outfile, node, client_num)
+                        b_size, cnt_c, object_id, layout, outfile, node, client_num
+                    )
                     self.motr_obj.cp_update_cmd(
-                        b_size=b_size, count=cnt_u,
-                        obj=object_id, layout=layout,
-                        file=infile, node=node, client_num=client_num, offset=offset)
-                    self.motr_obj.cat_cmd(b_size, cnt_c, object_id, layout, outfile, node,
-                                          client_num)
+                        b_size=b_size,
+                        count=cnt_u,
+                        obj=object_id,
+                        layout=layout,
+                        file=infile,
+                        node=node,
+                        client_num=client_num,
+                        offset=offset,
+                    )
+                    self.motr_obj.cat_cmd(
+                        b_size, cnt_c, object_id, layout, outfile, node, client_num
+                    )
                     self.motr_obj.md5sum_cmd(infile, outfile, node, flag=True)
                     self.motr_obj.unlink_cmd(object_id, layout, node, client_num)
 
             logger.info("Stop: Verify multiple m0cp/cat operation")
+        return True  # Todo: return status to be worked as per responses
```

Comment:
We won't require return statement as in the individual function of m0cp/m0cat/md5sum,unlink have assert.